tests/ops/functions/test_equal.py

```diff
@@ -1199,35 +1203,83 @@ def test_unsupported_object_type_not_implemented(self):
             qml.equal(dev, dev)
 
 
-class TestControlledComparisons:
+class TestSymbolicOpComparison:
     WIRES = [(5, 5, True), (6, 7, False)]
 
+    BASES = [
+        (qml.PauliX(0), qml.PauliX(0), True),
+        (qml.PauliX(0) @ qml.PauliY(1), qml.PauliX(0) @ qml.PauliY(1), True),
+        (qml.CRX(1.23, [0, 1]), qml.CRX(1.23, [0, 1]), True),
+        (qml.CRX(1.23, [1, 0]), qml.CRX(1.23, [0, 1]), False),
+        (qml.PauliY(1), qml.PauliY(0), False),
+        (qml.PauliX(1), qml.PauliY(1), False),
+        (qml.PauliX(0) @ qml.PauliY(1), qml.PauliZ(1) @ qml.PauliY(0), False),
+    ]
+
+    PARAMS = [(1.23, 1.23, True), (5, 5, True), (2, -2, False), (1.2, 1, False)]
+
+    def test_mismatched_arithmetic_depth(self):
```

Comment:
Just double-checking: these tests were moved in the file untouched, correct?

Reply:
Yes, just copy-pasted